libs/bits/bit_array_test.go

```diff
@@ -320,3 +320,45 @@ func TestBitArrayFromProto(t *testing.T) {
 		}
 	}
 }
+
+var sink interface{}
```

Comment:
this should definitely be in the scope of the benchmark

Reply:
Nope! It should be a global, to ensure a write barrier is erected and to prevent what we are trying to avoid which is optimizing away the code.